buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -171,8 +171,7 @@ class BuildPlugin implements Plugin<Project>  {
                     project.rootProject.ext.hadoopClient = ["org.apache.hadoop:hadoop-client:$version"]
                     println "Using Apache Hadoop on YARN [$version]"
                     break
-
-                default:
+                case "hadoopStable":
```

Comment:
We should add a `default:` case and throw an error, so we don't accidentally introduce leniency in the future